engine/phone.asm

```diff
@@ -165,7 +165,7 @@ CheckPhoneContactTimeOfDay: ; 900ad (24:40ad)
 
 	callba CheckTime
 	pop af
-	and (1 << MORN) + (1 << DAY) + (1 << NITE)
+	and MORN + DAY + NITE
```

Comment:
`MORN | DAY | NITE`

Reply:
Done; IIRC there are other places which use + for combining masks, will keep an eye out for them.